### PR TITLE
Cancel simulation and get user simulations

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

File: yaptide/celery/tasks.py [hunk 1]
```diff
@@ -192,3 +192,11 @@ def sh12a_simulation_status(path_to_file: str):
             }
         }
         return sim_info
+
+
+@celery_app.task
+def cancel_simulation(task_id: str) -> dict:
+    """Task responsible for canceling simulation in progress"""
```
Comment (grzanka) [inline yaptide/celery/tasks.py:199]: Is it sending SIGTEM singal to python process and then to the shieldhit executable ?

**Comment by grzanka** [comment]
Is it possible to obtain output from cancelled simulations ?

File: yaptide/celery/tasks.py (hunk 1 above)
Comment (p1003) [reply yaptide/celery/tasks.py:199]: No, it does not I need to change it

**Comment by p1003** [comment]
> 

Currenlty it is not and might not be added in this pull request

**Review by grzanka — DISMISSED** [review]
LGTM

File: yaptide/celery/tasks.py (hunk 1 above)
Comment (grzanka) [reply yaptide/celery/tasks.py:199]: What about changing that in separete PR ?

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**